src/map/Map.js

```diff
@@ -16,7 +16,9 @@ L.Map = L.Evented.extend({
 		fadeAnimation: true,
 		trackResize: true,
 		markerZoomAnimation: true,
-		maxBoundsViscosity: 0.0
+		maxBoundsViscosity: 0.0,
+		zoomSnap: 1,
+		zoomDelta: 1

```

Comment:
Any user interaction for the zoom level (keyboard, scrollwheel, zoom buttons) will change zoom by this amount.